releasenotes/notes/add-pre-commit-hooks-07ce3b82bbe1d7a3.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Add playbook to install `pre-commit` hooks and register them with git.
```

Comment:
I think you need double backticks.

Reply:
Looking at the docs the examples use single back ticks https://docs.openstack.org/reno/latest/user/examples.html#input-file

```
fixes_command_line:
  - |
    This is a subsection. It requires setting `sections` in
```

Reply:
> Looking at the docs the examples use single back ticks https://docs.openstack.org/reno/latest/user/examples.html#input-file
> 
> ```
> fixes_command_line:
>   - |
>     This is a subsection. It requires setting `sections` in
> ```

I think they got it wrong… And it isn't rendered so we don't see the result. Single backtick produces italic text with reno.

See https://docutils.sourceforge.io/docs/user/rst/quickref.html for details.